app-antivirus/fangfrisch/files/conf

```diff
@@ -0,0 +1,22 @@
+# See https://rseichter.github.io/fangfrisch/ for detailed documentation
+
+[DEFAULT]
+db_url = sqlite:////var/lib/fangfrisch/db.sqlite
```

Comment:
Why four slashes? I can get two as part of `sqlite://`, one as part of path… but why the fourth one?

Reply:
Four slashes are required when specifiying an absolute path for the `sqlite:` URL schema, see [SQLAlchemy documentation](https://docs.sqlalchemy.org/en/13/core/engines.html).